pkg/api/validation/validation.go

```diff
@@ -219,7 +219,10 @@ func ValidateObjectMetaUpdate(old, meta *api.ObjectMeta) errs.ValidationErrorLis
 	if len(meta.UID) == 0 {
 		meta.UID = old.UID
 	}
-	if meta.CreationTimestamp.IsZero() {
+	// ignore changes to timestamp
+	if old.CreationTimestamp.IsZero() {
+		old.CreationTimestamp = meta.CreationTimestamp
+	} else {
```

Comment:
meta has to have the same creation timestamp as the new one, otherwise the latter check fails.  The problem here is both are set, and are different.  This is just about ensuring that "meta" (what we are going to write) is correctly set to *something*.  If old.CreationTimestamp is zero, it seems strictly better to have it set to something (vs remaining empty)